dataland-frontend/src/components/general/UserProfileDropDown.vue

```diff
@@ -1,13 +1,13 @@
 <template>
   <div
     @click="toggleDropdownMenu"
-    class="max-w-full max-h-full flex justify-content-center d-drop-down-toggle"
+    class="flex align-items-center d-drop-down-toggle"
```

Comment:
Fürs CSS-Learning:
Warum nicht auch zusätzlich `justify-content-center` drin gelassen, um horizontal zu zentrieren?

Reply:
Das brauchen wir hier nicht, da der UserProfileDropDown container genau nur die benötigte Größe hat (also links und rechts kein leerraum ist, in den man das hinein zentrieren müsste). Wenn dann, wäre es hier auch wahrscheinlich sinnvoller, das ganze ans ende (als nach rechts) zu schieben, da das drop down menü ja nicht zentriert, sondan auf der rechten Seite erscheinen soll.